cypress/integration/finance/fiscalYears/fiscalYears.create.spec.js

```diff
@@ -6,7 +6,7 @@ import testType from '../../../support/dictionary/testTypes';
 import FinanceHelp from '../../../support/fragments/finance/financeHelper';
 import devTeams from '../../../support/dictionary/devTeams';
 
-describe('ui-finance: Fiscal Year creation', () => {
+describe('ui-finance: Fiscal Year creation', { retries: 3 }, () => {
```

Comment:
Why do you add this retry in so many tests?

Reply:
I want to check now that it has stood for a few runs. The tests just in the wound fall completely different all the time